scripts/update_qm_list.py

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+import hera_mc.mc as mc
+
+parser = mc.get_mc_argument_parser()
+args = parser.parse_args()
+db = mc.connect_to_mc_db(args)
+session = db.sessionmaker()
+
+metric_list = copy(ant_metric_list)
```

Comment:
I don't see how this script can work since nothing defines the variables `copy`, `ant_metric_list`, `firstcal_metric_list`, etc.